frontend/src/types/tab.ts

```diff
@@ -8,9 +15,18 @@ export type ExecuteOption = {
   explain: boolean;
 };
 
+export type Connection = {
+  projectId: ProjectId;
+  instanceId: InstanceId;
+  databaseId: DatabaseId;
+  tableId: TableId;
+  tableName: string;
```

Comment:
As we have `tableId`, do we really need the `tableName`?

Reply:
It's awkward that we don't have an API endpoint to get a table entity by tableId.
But I consider we may get it from tableList. Let me take another look.

Reply:
Updated. Removed `tableName` field. Find the table by `databaseId` and `tableId` in the `tableStore.tableListByDatabaseId` map.